shared/wallets/trustline/index.tsx

```diff
@@ -0,0 +1,162 @@
+import * as I from 'immutable'
+import * as React from 'react'
+import * as Styles from '../../styles'
+import * as Kb from '../../common-adapters'
+import * as Types from '../../constants/types/wallets'
+import Asset from './asset-container'
+
+type BodyProps = {
+  totalAssetsCount: number
+  errorMessage?: string
+  acceptedAssets: I.List<Types.TrustlineAssetID>
```

Comment:
we typically avoid passing immutable structure to the components

Reply:
Cool; I'll address this in the next PR https://github.com/keybase/client/pull/17992